UPGRADING.rst

```diff
@@ -11,6 +11,42 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#2246 Add deployment incarnation counter
+========================================
+
+See upgrading instructions for #2143 below.
+
+
+#2143 Merge service accounts for indexer and service
+====================================================
+
+1.  Before upgrading to this commit, run ::
+
+        make -C terraform validate
```

Comment:
This is missing stuff. And the `-C terraform` followed by `(cd terraform` is a bit weird since both temporarily cd into that directory.

I don't know if this is correct but use it for inspiration:

```diff
Index: UPGRADING.rst
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- UPGRADING.rst	(revision 90c1b03c15994e1601105df9252be3cd86b05bb0)
+++ UPGRADING.rst	(date 1603338157551)
@@ -22,8 +22,12 @@
 
 1.  Before upgrading to this commit, run ::
 
-        make -C terraform validate
-        (cd terraform && terraform destroy -target=google_service_account.indexer)
+        source environment
+        _select foo
+        _preauth
+        cd terraform
+        make validate
+        terraform destroy -target=google_service_account.indexer
+        cd ..
 
 2.  Upgrade to this commit or a later one.
 
```